rv-monitor-rt/src/main/java/com/runtimeverification/rvmonitor/java/rt/util/TraceDB.java

```diff
@@ -24,10 +24,14 @@ public class TraceDB {
 
     public TraceDB() {
         this.connection = getConnection();
+ 	 System.setProperty("dbFilePath", "/tmp/tracedb");
```

Comment:
1. We should later allow the user to supply their own `dbFilePath`. Please open an issue for that.
2. `/tmp/tracedb` is not platform independent; please fix
3. Please add a comment before you set any property in both constructors to say why you are setting the system properties.